src/components/RP_GettingStarted.component

```diff
@@ -0,0 +1,82 @@
+<apex:component controller="RP_GettingStartedController">
+
+    <div class="slds-m-around_x-small">
+
+        <h1 class="slds-text-heading_large slds-p-around_medium">{!$Label.RP_GettingStarted}</h1>
+
+    </div>
+
+    <div class="slds-m-vertical_medium slds-media slds-media_responsive">
+
+        <div class="slds-p-horizontal_xx-large slds-size--1-of-1 slds-large-size--1-of-3">
+
+            <img class="slds-p-vertical_medium slds-align_absolute-center" id="trailheadImage" src="{!URLFOR($Resource.RP_Resources, 'Resources/images/Image_Trailhead.png' )}" width="150" height="60"/>
```

Comment:
you have created an additional static resource for your use, which makes sense, but in general we've put all of our static resources into a single bundle, CumulusStaticResources.resource, which allows us to change the what  static resources we need, without being constrained by the manage package not allowing one to delete a static resource after it's been packaged once.  @kbromer @jlantz either of you recall other reasons why we went with one?

The sources that make up the static resource need to be included in the project under StaticResourceSources, which you can the zip together to create a new CumulusStaticResources.resource to be part of your pull request.